src/components/VersionSelect/index.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Form } from 'react-bootstrap';
+
+import { fakeVersions } from '../../test-helpers';
+
+import VersionSelect from '.';
+
+describe(__filename, () => {
+  const render = (props = {}) => {
+    const allProps = {
+      label: 'select a version',
+      listedVersions: [
+        {
+          ...fakeVersions[0],
+          channel: 'listed',
+        },
+      ],
+      unlistedVersions: [
+        {
+          ...fakeVersions[1],
+          channel: 'unlisted',
+        },
+      ],
+      ...props,
+    };
+
+    return shallow(<VersionSelect {...allProps} />);
+  };
+
+  it('renders a select form control', () => {
+    const root = render();
+
+    expect(root.find(Form.Control)).toHaveLength(1);
+    expect(root.find(Form.Control)).toHaveProp('as', 'select');
+  });
+
+  it('renders a label', () => {
+    const label = 'some label';
+    const root = render({ label });
+
+    expect(root.find(Form.Label)).toHaveLength(1);
+    expect(root.find(Form.Label)).toIncludeText(label);
+  });
+
+  it('renders two lists of versions', () => {
+    const listedVersions = [
+      {
+        id: 123,
+        channel: 'listed',
+        version: 'v1',
+      },
+    ];
+    const unlistedVersions = [
+      {
+        id: 456,
+        channel: 'unlisted',
+        version: 'v2',
+      },
+    ];
+
+    const root = render({ listedVersions, unlistedVersions });
+
+    expect(root.find('optgroup').at(0)).toHaveProp('label', 'Listed');
```

Comment:
Instead of `.at(0)` and `.at(1)`, I suggest adding classes for each (like `.listedContro`) and using those in your query.